tools/check-imports.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import glob
+import os
+import re
+import sys
+
+
+def do_exist(file, lines, imported):
```

Comment:
@addaleax I don't know how I missed that. Thanks for catching it :-) Fixed it now.